src/common/download.js

```diff
@@ -1,7 +1,10 @@
 export const download = (fileName, value) => {
   const element = document.createElement("a");
-  element.setAttribute("download", fileName);
-  element.setAttribute("href", `data:text/plain,charset=utf-8,${encodeURIComponent(value)}`);
+  element.target = "_blank";
+  element.download = fileName;
+  element.href = `data:text/plain,charset=utf-8,${encodeURIComponent(value)}`;
```

Comment:
The comma after `data:text/plain` should be a semi-colon otherwise we get `charset=utf-8,` included as part of the content of the file being downloaded. Looks like this is a long-standing bug.

Reply:
This is now fixed.